extensions/amp-access/0.1/test/test-amp-access-client.js

```diff
@@ -53,13 +53,35 @@ describe('AccessClientAdapter', () => {
           .equal('https://acme.com/a?rid=READER_ID');
       expect(adapter.pingbackUrl_).to
           .equal('https://acme.com/p?rid=READER_ID');
+      expect(adapter.authorizationTimeout_).to
+          .equal(3000);
       expect(adapter.getConfig()).to.deep.equal({
         authorizationUrl: 'https://acme.com/a?rid=READER_ID',
         pingbackUrl: 'https://acme.com/p?rid=READER_ID',
+        authorizationTimeout: 3000,
       });
       expect(adapter.isAuthorizationEnabled()).to.be.true;
     });
 
+    it('should set authorization timeout if provided', () => {
+      let adapter;
+
+      validConfig['authorizationTimeout'] = 5000;
+      adapter = new AccessClientAdapter(window, validConfig, context);
+      expect(adapter.authorizationTimeout_).to.equal(5000);
+
+      validConfig['authorizationTimeout'] = 1000;
```

Comment:
I wanted to stub `getMode` here to verify that in production 1s is 1s, and 5s is cut to 3s, but couldn't wrap a single function with Sinon.